partcad/src/partcad/part_config.py

```
@@ -53,3 +54,8 @@ def normalize(name, config):
                     param_value["type"] = "float"
 
         return ShapeConfiguration.normalize(name, config)
+
+    @staticmethod
+    def get_manufacturing_data(part):
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `get_manufacturing_data`

(ANN205)